docs/index.md

```diff
@@ -19,7 +19,8 @@ Persona Essentials is a set of miscellaneous improvements for Persona 3 Portable
 - Support for injecting custom music into the game without extracting files.  
 - Support for hot reload (update files on the fly without game reboot).  
 - Automatically merges most .tbl files.  
-- Automatically merges most PAK (.bin) files.
+- Automatically merges PAK (.bin) files.
```

Comment:
These lines need double spaces at the end of them in order to produce a line break in the wiki/site version.